README.md

```diff
@@ -38,116 +38,53 @@ Here are a few key pages you might be interested in:
 
 ## Quickstart
 
-TigerBeetle is easy to run with or without Docker, depending on your
-preference. First, we'll cover running the [Single
-Binary](#single-binary). And below that is how to run [with
-Docker](#with-docker).
+First, grab a prebuilt copy of TigerBeetle.
 
-### Single Binary
+On macOS/Linux:
 
-Install TigerBeetle by grabbing the latest release from
-GitHub.
-
-x86_64 and aarch64 builds are available for macOS and Linux. Only
-x86_64 builds are available for Windows.
-
-For example:
-
-```bash
-$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2023-03-27-weekly/tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
-$ unzip tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
-$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle # On Windows, add $(pwd) to $env:PATH instead.
-$ tigerbeetle version --verbose | head -n6
-TigerBeetle version experimental
-
-git_commit="55c8fdf1f52c7a174d1bc9d9785cf4e327cae182"
-
-build.mode=Mode.ReleaseSafe
-build.zig_version=0.9.1
+```console
+$ git clone https://github.com/tigerbeetledb/tigerbeetle; ./tigerbeetle/scripts/bootstrap.sh
```

Comment:
That's a big improvement over the current quick start!

Not sure if that's the right long-term shape --- I expect that "installing tigerbeetle on CI" shouldn't involve clonning the repo. But it's not line we should solve each and every problem right now.

Reply:
Lots of systems marketing one-liner is not the way you are suggested to install in CI fwiw. We'll want to be in all the package managers eventually.

Reply:
Huge leap forward!

Let's do:

`git clone https://github.com/tigerbeetledb/tigerbeetle; ./tigerbeetle/bootstrap.sh`

To move `bootstrap` to root for its importance.

Reply:
Can do.